### PR TITLE
i18n

### PR DESCRIPTION
**Pull request by benoitchantre**
Makes the plugin ready for localization.

### CODE REVIEW

File: maintenance-mode.php [hunk 1]
```diff
@@ -30,3 +32,15 @@
...
+ * Localize plugin
+ *
+ * @since 0.1.1
+ * @uses load_plugin_textdomain
+ * @return void
+ */
+function vip_maintenance_mode_load_plugin_textdomain() {
+	load_plugin_textdomain( 'maintenance-mode', FALSE, basename( dirname( __FILE__ ) ) . '/languages/' );
+}
+add_action( 'plugins_loaded', 'vip_maintenance_mode_load_plugin_textdomain' );
```
Comment (mjangda) [inline maintenance-mode.php]: `init` may be a better hook here.

File: template-maintenance-mode.php [hunk 2]
```diff
@@ -4,8 +4,8 @@
 	<?php wp_head(); // allow for remote-login on mapped domains ?>
 </head>
 <body>
-	<h1>Howdy!</h1>
-	<p>We're just freshening things up a bit; back in a few!</p>
+	<h1><?php esc_html_e( 'Howdy!', 'maintenance-mode' ); ?></h1>
+	<p><?php esc_html_e( "We're just freshening things up a bit; back in a few!", 'maintenance-mode' ); ?></p>
```
Comment (mjangda) [inline template-maintenance-mode.php]: Better to use single quotes wherever possible.

**Review by mjangda — CHANGES_REQUESTED** [review]
Two small changes, otherwise looks good.

File: maintenance-mode.php (hunk 1 above)
Comment (benoitchantre) [reply maintenance-mode.php]: Thanks, this is now updated. I followed the example in the [Plugin Handbook](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/). In the Codex and in the Code Reference, the `init` hook is used.

**Comment by benoitchantre** [comment]
@mjangda I think it would make sense to add the language attribute and the charset in this PR. Ok with that?

**Comment by mjangda** [comment]
Yep, sounds like a good idea.

**Review by mjangda — APPROVED** [review]
Looks great, thank you!

**Comment by benoitchantre** [comment]
Thanks for the review.

**Pull request merged**